storage/src/checker/mod.rs

```diff
@@ -374,12 +377,12 @@ mod test {
         err,
         CheckerError::HashMismatch {
             address,
-            partial_path,
+            path,
```

Comment:
Debugging this would have been easier if we constructed the error and used assert_eq! on them, as the debug output would highlight the difference.